### ISSUE TITLE
Can't read purple text in item popup

### ISSUE THREAD

**Issue by aJamDonut** [aJamDonut/ltc_bugs#23]
# Issue

Fill in the below to your best ability, missing one of the sections is okay!

**Game Version:** 0.9.4.79
**Reported by:** MachineLordZero
**Platform:** Steam

## Description

Can't read the text well

## Expected

Can read it

![image](https://github.com/user-attachments/assets/83b8f076-abfd-4bc6-bba6-88036bf06ae6)

## Actual

My eyes burn

## Replication steps

Burn your eyes reading the text

**Comment by aJamDonut** [comment]
Changed colors to be higher contrast 0.9.5
